pygmt/src/grdhisteq.py

```diff
@@ -211,9 +211,13 @@ def equalize_grid(
         with GMTTempFile(suffix=".nc") as tmpfile:
             if isinstance(outgrid, str):
                 output_type = "file"
-            else:
+            elif outgrid is None:
                 output_type = "xarray"
                 outgrid = tmpfile.name
+            else:
+                raise GMTInvalidInput(
+                    "Must specify 'output_type' either as 'numpy', 'pandas' or 'file'."
```

Comment:
```suggestion
                    "Must specify 'outgrid' as a string or None."
```

Reply:
Thanks for catching this copy/paste error.